lib/Languages/Galach/Values/Token/Range.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace QueryTranslator\Languages\Galach\Values\Token;
+
+use QueryTranslator\Languages\Galach\Tokenizer;
+use QueryTranslator\Values\Token;
+
+/**
+ * Word term token.
+ *
+ * @see \QueryTranslator\Languages\Galach\Tokenizer::TOKEN_TERM
+ */
+final class Range extends Token
+{
+    /**
+     * Holds domain string.
+     *
+     * @var string
+     */
+    public $domain;
+
+    /**
+     * @var string
+     */
+    public $rangeFrom;
+
+    /**
+     * @var string
+     */
+    public $rangeTo;
+
+    /**
+     * @param string $lexeme
+     * @param int    $position
```

Comment:
No alignment in PHPDoc, it unnecessarily adds to diff when updated.

Reply:
I just run the php-cs-fixer with the current settings, I guess it was never run before on the code.
I guess it is a good thing to comply to a CS here :)